var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -91,8 +91,10 @@ class Openmpi(AutotoolsPackage):
     variant(
         'fabrics',
         default=None if _verbs_dir() is None else 'verbs',
-        description='List of fabrics that are enabled',
-        values=('psm', 'psm2', 'pmi', 'verbs', 'mxm'),
+        description=("List of fabrics that are enabled ('rdma' enables "
```

Comment:
Verbs layer calls it "providers". Typically each interconnect vendor provides one for their hardware.  Full least providers is here https://github.com/linux-rdma/rdma-core/tree/master/providers